pootle/templates/editor/_toolbar.html

```diff
@@ -4,7 +4,9 @@
   <div class="unit-sort">
     <select id="js-filter-sort">
       <option selected value="default">{% trans "Default order" %}</option>
+      {% if display_priority %}
       <option value="priority">{% trans "Decreasing priority" %}</option>
```

Comment:
I agree that is clearer. Will wait for input anyway. That simple change can be landed independently of this PR since it affects code unchanged by this PR.